StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1115ParameterMustFollowComma.cs

```diff
@@ -319,12 +333,13 @@ private static void AnalyzeParameterList(SyntaxNodeAnalysisContext context, Base
 
         private static bool IsValidTrivia(SyntaxTrivia trivia)
         {
+            if (trivia.IsDirective)
+            {
+                return true;
+            }
```

Comment:
This check is probably not correct, as it will also mark `#region` / `#endregion` as valid, but I think they are not.